interview-questions.md

```diff
@@ -10,46 +10,100 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 Your answer:
 
+   Html and JSX have many similarities with a few main differences. One of the main difference is JSX stands Javascript Syntax Extension or  in developer slang Javascript Xml . If you know about Javascript it is its own language . JSX coincides with html and that’s where us developers get nick name javascript XML  from. Another difference is that html is supported by all browsers. JSX on the other hand had a different focus and JSX main focus is to incorporate  javascript so the browser can understand Javascript in html language . This process is as easy as coding javascript in curly braces{}. When coding javascript in to the curly braces {} this will take all of the Javascript text and turn it into html reading on the back end so it will be able to display in the browser for human read ability. This process only happens under one condition,  which is JSX must be written inside of fragments and fragments allows you to take all of your components and return multiple elements. Fragments also allows us to group a list of children without adding a million lines of code.
+
+
+
+
+
 Researched answer:
 
+HTML is a Hypertext Markup Language, the standard language for documents designed to display and view on the web in a web browser. JSX (JavaScript + XML) is an extension of JavaScript that permits you to write down HTML directly within JavaScript, which features a few benefits of creating your code more readable and exercising the complete power of JavaScript within HTML. JSX is in some ways almost like HTML, but it does accompany certain differences. Since JSX isn’t a legitimate JAvascript code, it must be compiled into JS with a tool like Babel or similar. JSX is basically a fundamental way of how you define React components.JSX stands for JavaScript XML. It is a syntax extension for JavaScript that lets you use HTML tags right inside your JavaScript file. Instead of creating, configuring, and appending your HTML tags through JavaScript objects, you can create the elements using an XML-like syntax that will generate the DOM elements for you behind the scenes. Below is an example of JSX
+
+const App = <h1>Welcome to GeeksforGeeks</h1>;
```

Comment:
So please remember, even stitching together answers from multiple sites is still plagiarizing. 

[JSX (JavaScript + XML) is an extension of JavaScript that permits you to write down HTML directly within JavaScript, which features a few benefits of creating your code more readable and exercising the complete power of JavaScript within HTML.](https://www.geeksforgeeks.org/what-are-the-differences-between-jsx-and-html/#:~:text=Introduction%20to%20JSX%3A%20JSX%20(JavaScript,power%20of%20JavaScript%20within%20HTML. )

[What is JSX Exactly?
JSX stands for JavaScript XML. It is a syntax extension for JavaScript that lets you use HTML tags right inside your JavaScript file. ](https://www.webtips.dev/javascript-vs-jsx/)


[ h1 Welcome to GeeksforGeeks /h1 ;](https://www.geeksforgeeks.org/reactjs-functional-components/)